src/utils/urlUtils.ts

```diff
@@ -38,7 +38,12 @@ export function getDataFile(): string {
   }
   return data;
 }
+export function isItLastAssessment(): boolean {
 
+  let pathParams = getPathName();
+  let requiredScore = Number(pathParams.get("requiredScore"));
+  return requiredScore >= 640
```

Comment:
This is wrong right here, @amitsinghsutara 
We can't just compare the required score to 640 which is a hardcoded value.

Once we get the score and we are about to send the completed event we have to compare the requiredScore from the path params to the assessment score for Letter Sounds that the user gets at the end of Letter Sounds and send a bool if the nextAssessment is null or not 🙌🏽